src/main/java/ai/elimu/web/SignOnControllerDiscord.java

```diff
@@ -132,10 +132,25 @@ public String handleCallback(HttpServletRequest request) throws IOException, Int
                 String id = String.valueOf(idAsLong);
                 contributor.setProviderIdDiscord(id);
             }
+            private static final String DEFAULT_AVATAR_URL = "https://e7.pngegg.com/pngimages/84/165/"
```

Comment:
@venkatesh2k3 The `imageUrl` is not a required field: https://github.com/elimu-ai/webapp/blob/main/src/main/java/ai/elimu/model/contributor/Contributor.java#L57

Therefore, we don't need to fallback to a default image. Instead, it would be simpler (less code) to just leave it empty.